src/App/App.js

```diff
@@ -29,9 +29,17 @@ class App extends Component {
         </nav>
         <Route path="/login" render={() => <Login />} />
```

Comment:
We'll see in the Router lesson that we can often simply use component={<Login />} instead of render={() => <Login />} when we do not need to pass in any props!